### PR TITLE
Gitlab merge-request review support

### PR DESCRIPTION
**Pull request by asfaltboy**
Hi guys, had some time to hack this up today; which, if accepted, would be the beginning of implementing PR #363

### CODE REVIEW

File: github/commands/commit.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 from ...common import util
 
 
-class GsShowGithubIssuesCommand(TextCommand, GitCommand, git_mixins.GithubRemotesMixin):
+class GsGithubShowGithubIssuesCommand(TextCommand, GitCommand, git_mixins.GithubRemotesMixin):
```
Comment (stoivo) [inline github/commands/commit.py]: Should there classes have Github twice in it's name

**Comment by stoivo** [comment]
A good start. I think we can merge this into dev.

File: github/commands/commit.py (hunk 1 above)
Comment (asfaltboy) [reply github/commands/commit.py]: Good point, I think I caught all of these in the command names, this one may have slipped through.

**Comment by asfaltboy** [comment]
Thanks @stoivo for the review, I added a couple more commits, one that resolves the GsGithub naming issue mentions, and another that implements 3 of the "open on web" commands (file, repo home and issues). I also went ahead and marked the completed features in #364 todo list.

**Comment by stoivo** [comment]
If you are not going to do anything more here for the next couple of weeks I would like to merge this.

**Comment by asfaltboy** [comment]
Yes, I'm out of time for now, until next time... though I may need to add bitbucket support next :)

**Pull request merged**